modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -422,6 +422,20 @@ private void postProcessEnumRefs(final Map<String, Object> models) {
                         }
                     }
 
+                    if (operation.examples != null){
+                        for (Map<String, String> example : operation.examples)
+                        {
+                            for (Map.Entry<String, String> entry : example.entrySet())
+                            {
+                                // Replace " with \", \r, \n with \\r, \\n
+                                String val = entry.getValue().replace("\"", "\\\"")
+                                    .replace("\r","\\r")
```

Comment:
There are a few issues with this approach that I see…

1. This will replace unexpected results. For instance, what if my example was:
   ```
    { 
        "example_commands": [
            "\run", "\nip", "\yap", "\rollover"
        ],
        "fast_commands": [
            "\\run", "\\rollover"
         ],
        "omg_slow_down_puppy": [
             "\\\run", "\\\rollover"
         ],
        "search_command" : "\\"
    }
    ````
2. This will lead to failed compilation if we don't escape other things like double-quotes and format placeholders like `{0}`, `{0,15}`, `{0:t}`, etc.
3. afiak this will result in multiline strings, regardless of how many times you escape these characters. I could be wrong on that, though.

The amount of effort to resolve this is, I don't think, worth the benefit. 

Some of this could be resolved by replacing only on boundaries, for example a regex like `\b\\r\b` which would catch probably most cases outside of text occurrences:

```
this\r"matches
this\rdoesn't
```

Reply:
Great suggestions @jimschubert!
On the double quotes.. I do replace these as well..

I'm not that fluent in Java..
How would I rewrite the code to do regex replace instead?

Thanks!